src/Command/ApiCompare.php

```diff
@@ -0,0 +1,83 @@
+<?php
+declare(strict_types=1);
+
+namespace Roave\ApiCompare\Command;
+
+use Roave\ApiCompare\Comparator;
+use Roave\ApiCompare\Factory\DirectoryReflectorFactory;
+use Roave\ApiCompare\Formatter\SymfonyConsoleTextFormatter;
+use Roave\ApiCompare\Git\CheckedOutRepository;
+use Roave\ApiCompare\Git\PerformCheckoutOfRevision;
+use Roave\ApiCompare\Git\Revision;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ApiCompare extends Command
+{
+    /** @var PerformCheckoutOfRevision */
```

Comment:
Please expand like the property below

Reply:
Oh, yeh the CS will eventually be this (non-expanded)